lib/routes/objects.js

```diff
@@ -802,5 +803,65 @@ module.exports =
           res.status(204);
         }
     });
-    /* eslint-enable max-statements */
+
+    fastify.route({
+      method: 'GET',
+      url: '/logs/:name/:env',
+      schema: {
+        params: {
+          type: 'object',
+          properties: {
+            name: { type: 'string' },
+            env: { type: 'string' }
+          }
+        },
+        response: {
+          200: {
+            type: 'array',
+            items: {
+              type: 'object',
+              properties: {
+                version: { type: 'string' },
+                releaseTimestamp: { type: 'number' },
```

Comment:
If `releaseDate` is an ISO8601 datetime, then `releaseTimestamp` is unnecessary or duplicative information and we can drop it. (My preference will always be for ISO 8601 datetime strings over unix epochs.)